src/app/backend/integration/metric/manager.go

```diff
@@ -40,7 +41,9 @@ type MetricManager interface {
 	EnableWithRetry(id integrationapi.IntegrationID, period time.Duration)
 	// List returns list of available metric related integrations.
 	List() []integrationapi.Integration
-	// ConfigureHeapster configures and adds heapster to clients list.
+	// ConfigureSidecar configures and adds sidecar to clients list.
+	ConfigureSidecar(host string) MetricManager
+	// ConfigureSidecar configures and adds sidecar to clients list.
```

Comment:
I think you want this comment to be for the ConfigureHeapster function.

Reply:
You are correct. :) Thanks!

Reply:
Fixed. :)